.yarn/sdks/eslint/bin/eslint.js

```diff
@@ -2,7 +2,7 @@
 
 const {existsSync} = require(`fs`);
 const {createRequire, createRequireFromPath} = require(`module`);
-const {resolve} = require(`path`);
+const {resolve, dirname} = require(`path`);
```

Comment:
Is there a reason these are all in backtick-strings?

Reply:
This file is auto-generated, and that's yarn's styleguide.